### PR TITLE
feat: connection handling for client code and tests

### PR DESCRIPTION
**Pull request by fan711**
- added connection handling so client code and tests can properly close connections

- currently tests could not close database connections which leads to failing test suites

- a closeAllConnections method is exported to allow closing connections

### CODE REVIEW

**Review by rudolfix — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@wallfair.io/trading-engine",
-  "version": "0.1.15",
+  "version": "0.1.16",
```
Comment (SamirHodzic) [inline package.json:3]: We are not manully bumping versions on this library, it is automated on release action.

File: src/db/config.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ const config: ConnectionOptions = {
   username: process.env.POSTGRES_USER,
   password: process.env.POSTGRES_PASSWORD,
   database: process.env.POSTGRES_DB,
-  ssl: process.env.POSTGRES_DISABLE_SSL !== 'true',
+  // ssl: process.env.POSTGRES_DISABLE_SSL !== 'true',
```
Comment (SamirHodzic) [inline src/db/config.ts:10]: Why is this commented? We need to include ssl cert for pg connection.

File: src/index.ts [hunk 3]
```diff
@@ -21,3 +22,4 @@ export {
   Transactions,
   TransactionManager,
 } from './lib/modules';
+(async () => await initDb())();
```
Comment (SamirHodzic) [inline src/index.ts:25]: Why this? Not sure we kinda want lib automatically to try connecting without leaving track anyway in consumer codebase.

File: src/db/config.ts (hunk 2 above)
Comment (fan711) [reply src/db/config.ts:10]: this is a leftover from testing, I'll revert that

File: src/index.ts (hunk 3 above)
Comment (fan711) [reply src/index.ts:25]: I had quite some issues with connection handling in tests, I'll update cliend code to handle this

**Pull request merged**